TypeCobol/Compiler/Scanner/MultilineScanState.cs

```diff
@@ -141,6 +146,7 @@ public MultilineScanState Clone()
             MultilineScanState clone = new MultilineScanState(InsideDataDivision, InsideProcedureDivision, InsidePseudoText, InsideSymbolicCharacterDefinitions,
                 InsideFormalizedComment, InsideMultilineComments, InsideParamsField, 
                 DecimalPointIsComma, WithDebuggingMode, EncodingForAlphanumericLiterals);
+            clone.InsideCopy = this.InsideCopy;
```

Comment:
No new constructor parameter ?

Reply:
Yes this property is explicitly set in CompilationDocuemnt.